4/Step 1/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Client } from "./Client";
+
+const client = new Client();
+
+client.setShipment({
```

Comment:
Please create all possible types of Shipments and attach screenshot after running them in conversations. Mock data with both valid and invalid zipcodes, different weights, etc.